### DISCUSSION TITLE
Inherited properties

### DISCUSSION THREAD

**Discussion by 3ch023** [adobe/helix-home#331] [Q&A]
In AEM, we use [inherited](https://developer.adobe.com/experience-manager/reference-materials/6-5/javadoc/com/day/cq/commons/inherit/InheritanceValueMap.html) page properties for many different use cases. With our move to Franklin/Milo, what is the alternative place to store these properties?

Some of these properties we could move to config in code and agree that Content Authors won't have control over them anymore.
But where to put the properties that Authors still should be able to control without a code change?

For example, Author wants to set 'checkoutRedirect' like this:
/website                                              - '/checkout/home'
/website/productA/                            - '/checkout/productA'
/website/productA/subproduct         - '/checkout/productA/promo'
/website/productB/                            - '/checkout/productB'

I've seen [metadata](https://www.hlx.live/developer/block-collection/metadata) being used for similar use cases. However, it doesn't feel like the right place for it because these properties are not metadata and shouldn't be in <meta> tags.

**Comment by honstar** [comment]
I think having an analogous mechanism to the already available bulk metadata (https://www.hlx.live/docs/bulk-metadata) would make sense. Paired with a local override mechanism, this would satisfy the requirements.

**Reply by sdmcraft** [reply]
In the bulk metadata you could specify globs for paths. For eg /folder/** for everything in folder and /folder/foo for a specific page
